en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
+- Any additional infrastructure costs incurred (i.e., if the infrastructure cost is over $100). 
 
-To search for real-time flight data, a client enters a search criteria, and searches through the web application. Upon receiving the request, the client application (web application) makes a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo then invokes the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API then invokes a database to get the information and passes it to the client.
+This plan does not require you to pay an annual fee. You can deploy up to 25 components (i.e., any combination of component types). 
 
-#### Glossary
+If the number of components you need to deploy exceeds 25 or if your annual spend exceeds $50,000, we recommend that you change your pricing plan to Enterprise.
 
-- <a class="pricing" id="flighStatusAPI">Flight Status API</a>: An external-facing, managed API on Choreo. This API exposes the Real-Time Flight API on Choreo as a managed API.
+**Enterprise** 
 
-- <a class="pricing" id="realtimeFlightAPI">Real-TIme Flight API</a>: An unmanaged, external-facing API hosted outside of Choreo.
+This plan allows you to pay an annual subscription of $50,000 or more (as agreed) and deploy up to 50 components (i.e., any combination of component types). If you need to deploy over 50 components, you can renegotiate with Choreo to pay an agreed amount per additional component or revise the annual subscription.
 
-- <a class="pricing" id="flightsDatabase">Flights Database</a>: A database that stores flight information.
+The following table summarizes the cost of each plan:
 
+ |                      | **Free**       | **Pay As You Go** | **Enterprise** |
+ |----------------------|----------------|-------------------|----------------|
+ | Subscription    | Free subscription  | Based on the number of components| Minimum $50k² for the annual commitment. The annual commitment includes 50x components (any combination of component types). |
+ | Infrastructure  | Up to $100 worth of infrastructure services free of charge. | Up to $100 worth of infrastructure services free of charge.<br/><br/> | Pass through or pay directly to Azure. |
 
-#### Get flight status
+The following table summarizes the usage of each plan:
 
-![Get flight status](../assets/img/references/api-proxy-example1.png)
+ |                      | **Free**       | **Pay As You Go** | **Enterprise** |
+ |----------------------|----------------|-------------------|----------------|
+ | **No of Projects**   | 1              | Unlimited         | Unlimited      |
+ | **No of Components** | A maximum of 5 | A maximum of 25 (recommended)  | Unlimited      |
+ | **No of Developers** | Unlimited      | Unlimited         | Unlimited      |
 
-Let’s see how the steps are calculated for the process to obtain the flight status:
+### Features
```

Comment:
Since all features are supported by all plans, shall we have that sentence as a common sentence?

Reply:
Although all features are supported, some features are supported with limitations based on the payment. I think having such a statement at the beginning can mislead.